modules/custom/d_content/templates/page--node.html.twig

```diff
@@ -0,0 +1,63 @@
+{#
+/**
+ * @file
+ * Theme override to display a contente page .
+ *
+ * @see template_preprocess_page()
+ */
+#}
+{% set classes = [
+  'navbar-wrapper',
+  header_layout == 'header_with_cta' ? 'mb-lg-2',
+] %}
+{% extends "page.html.twig" %}
```

Comment:
ok, a po co w ogole ten twig ląduje w d_content? mamy przeciez droopler_theme i w nim już page.html.twig, nie mozna tam tego dać?

Reply:
Przeniosłem wszystko do page.html.twig bo w sumie to nie potrzebnie dodane było to do innego szablonu.  Trzeba i tak nadpisywać szablon dla page a nie dla noda.